### ISSUE TITLE
Perl 5.28.1 on Ubuntu warns of future breakage

### ISSUE THREAD

**Issue by Corvan** [philpennock/sieve-connect#12]
```
Unescaped left brace in regex is deprecated here (and will be fatal in Perl 5.30), passed through in regex; marked by <-- HERE in m/^{ <-- HERE (\d+)\+?}\r?$/ at /usr/bin/sieve-connect line 926.
Unescaped left brace in regex is deprecated here (and will be fatal in Perl 5.30), passed through in regex; marked by <-- HERE in m/^{ <-- HERE (\d+)\+?}\r?$/ at /usr/bin/sieve-connect line 1509.
Unescaped left brace in regex is deprecated here (and will be fatal in Perl 5.30), passed through in regex; marked by <-- HERE in m/^{ <-- HERE \d+\+?}\r?\n?/ at /usr/bin/sieve-connect line 1525.
```

**Comment by it-philpennock** [comment]
I believe this was already fixed in PR 9, which was part of release v0.90.  What version did you see this in?

**Comment by philpennock** [comment]
I believe this was already fixed in PR 9, which was part of release v0.90. What version did you see this in?

**Comment by Corvan** [comment]
Ubuntu - Disco Dingo

**Comment by philpennock** [comment]
There is nothing I can do about what Ubuntu package.  If you have a support contract with Ubuntu, I suggest asking them to update the package.  I see that Disco is using `0.88-1`; 0.88 was released in 2016, 0.89 in 2017, 0.90 in March 2019.

Good luck!

**Comment by Corvan** [comment]
Allright, thx anyways
